.github/workflows/codeql.yml

```diff
@@ -0,0 +1,75 @@
+# For most projects, this workflow file will not need changing; you simply need
```

Comment:
Can we also configure to run it only for `aws` repository? Like this one here: https://github.com/aws/aws-lambda-builders/blob/develop/.github/workflows/build.yml#L43